server/util/roster-utils.ts

```diff
@@ -102,3 +105,11 @@ export async function getRosterHistoryForIndividual(edipi: string, unitId: numbe
     .addOrderBy('rh.change_type', 'DESC')
     .getMany();
 }
+
+export function getCsvHeaderForRosterColumn(column: RosterColumnInfo) {
+  return column.displayName;
```

Comment:
Do we need the mapping for the 'DoD ID' <-> 'edipi' translations and such?

Reply:
The `edipi` column's display name is just "DoD ID" now, so this is effectively doing that. From what I could tell, we were mapping to DoD ID everywhere, so we don't seem to need any additional logic for it.